cmd/fabric-ca-server/config.go

```diff
@@ -12,6 +12,8 @@ import (
 	"path/filepath"
 	"strings"
 
+	"regexp"
```

Comment:
Super nit: this should be grouped with the above imports

Reply:
Courtesy of GoImports. 🧹